src/components/QuizGameApi/index.jsx

```diff
@@ -0,0 +1,14 @@
+//first line to rendomlly picking a question
+const shuffleArray = (array) => [...array].sort(() => Math.random() - 0.5);
+//iam fetching data from a website called opentdb.com this website will give us a simple questions about the nature.
+export const fetchQuizQuestions = async (amount, difficulty) => {
+  const endpoint = `https://opentdb.com/api.php?amount=${amount}&category=17&difficulty=${difficulty}&type=multiple`;
```

Comment:
+1

Reply:
i tried it actually it didnot work ,it gave me an error

Reply:
1. if you say there is an error, please show the error
2. you are probably doing the incorrect thing, because it is not possible for what is being suggested not to work.

do the same thing as you did with `${amount}` but do it with the category. i won't tell you exactly what to name the variable because i want you to understand the problem that sule is talking about. if you still don't understand, please work with sule to understand what her comment meant

Reply:
@louisrli  i did exactly the same but didnot work 
![Screenshot from 2020-11-13 21-28-20](https://user-images.githubusercontent.com/67765520/99107971-b2180300-25f7-11eb-8a89-cb4141e495c4.png)

Reply:
the error it cant fetch the questions,ty peError: Cannot read property 'question' of undefined

Reply:
![Screenshot from 2020-11-13 21-33-44](https://user-images.githubusercontent.com/67765520/99108111-ebe90980-25f7-11eb-8fae-22dd2c9d0252.png)

Reply:
is that an error that says it can't fetch the questions? i'd
1. read your code carefully
2. think about what the error message says

and talk to sule or tarkan here about what to do.

Reply:
@louisrli  i fixed it ,i added extra space by mistake